src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -54,9 +61,22 @@ public PersonCard(Person person, int displayedIndex) {
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
+        if (person.getPhone() != null) {
+            phone.setText(person.getPhone().value);
```

Comment:
good to check for null case

Reply:
We already have a null case check for all of the variables in Person class, is this necessary?